src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -86,6 +86,25 @@ public function get($key)
     }
 
     /**
+     * Retrieve multiple items from the cache by key.
+     *
+     * Items not found in the cache will have a null value for the key.
+     *
+     * @param  array  $keys
+     * @return array
+     */
+    public function getMultiple(array $keys)
```

Comment:
I was thinking that as well.  Should the trait go in that same folder?  That seems to be the convention.